### DISCUSSION TITLE
Google Docs Bug

### DISCUSSION THREAD

**Discussion by Swiftburn** [birchill/10ten-ja-reader#1528] [General]
Hello, I wasn't sure where to post bug reports but here will do.

When I first downloaded 10ten at the beginning of 2023 I was able to look up Japanese words in google docs but recently that doesn't seem to be the case. 

On my main Windows PC I use Opera and Chrome, both of these do not let me use 10ten to look up Japanese. There seems to be some console errors but I'm unsure if its due to other addons or google docs itself.

![image_2023-12-28_015226841](https://github.com/birchill/10ten-ja-reader/assets/113044118/fd47bf49-04e6-4cab-9170-c65991487c4e)

![image_2023-12-28_015313075](https://github.com/birchill/10ten-ja-reader/assets/113044118/f4055f9d-8edd-4ac3-8020-c4369912874a)

I did find a work around that works for my main PC by using my Mac laptop. On my Mac I use Arc Browser and I found that I have 10ten Reader and Rikaikun, for some reason after enabling Rikaikun and then turning it off, 10ten will then work just fine like any other website. 

![image_2023-12-28_015548481](https://github.com/birchill/10ten-ja-reader/assets/113044118/db3aaf99-15de-40e3-94da-906862ad1228)

If this isn't the correct place to report bugs, please let me know what channel to go through to report bugs.

**Comment by birtles** [comment]
Hi! Thanks for the bug report! Yes, this is fine.

I'm really sorry about that bug. My guess is that it's a result of switching to MV3 in Chrome recently. It works fine for me in Firefox but I probably need to fix the CSP policy to work in Chrome again.

I'm planning to take the next few days off but hopefully I can fix it soon. Thanks for reporting and sorry for the trouble!

**Reply by Swiftburn** [reply]
Great! Just to reiterate this is an issue for both google chrome and opera browsers. It isn't a major bug and has a quick work around so please take your time. Hopefully others can point to this post as a temporary work around!

Happy Holidays!

**Reply by birtles** [reply]
Thanks! Yes, Opera and Chrome are using the same engine so I expect it affects them in the same way and also Edge too.

Happy holidays to you too!

**Reply by birtles** [reply]
@Swiftburn I've released version 1.17.1 for Chrome (the Edge review will take another week or so) which should hopefully fix this. Please let me know if it works for you!

**Reply by Swiftburn** [reply]
It seems to work as intended, thank you for your hard work making this amazing and wonderful app!
